mq/esworker_cloudtrail.py

```diff
@@ -257,6 +257,10 @@ def keyMapping(aDict):
         if 'utctimestamp' not in returndict:
             # default in case we don't find a reasonable timestamp
             returndict['utctimestamp'] = toUTC(datetime.now()).isoformat()
+        if 'type' not in returndict:
```

Comment:
I think we actually want to stomp on the type field on events here even if it exists specifically for cloudtrail events.

Reply:
Agreed.

Reply:
On line 222 we are converting "type" to category, which is why I put this later in the code.
How would you handle modifying this field so we aren't stomping on category?

Reply:
I've made a change, let me know if this is more what you were thinking.